src/xdocs/config_annotation.xml

```diff
@@ -1355,15 +1355,16 @@ package com.example.annotations.packageannotation; //ok
       </subsection>
 
       <subsection name="Examples" id="SuppressWarningsHolder_Examples">
-        <p>To prevent <code>FooCheck</code> violations from being reported write:</p>
-        <source>
-@SuppressWarnings("foo") interface I { }
-@SuppressWarnings("foo") enum E { }
-@SuppressWarnings("foo") InputSuppressWarningsFilter() { }
-        </source>
         <p>Some real check examples:</p>
         <p>This will prevent from invocation of the MemberNameCheck:</p>
         <source>
+&lt;module name=&quot;TreeWalker&quot;&gt;
+  &lt;module name=&quot;MemberName&quot;/&gt;
+  &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+&lt;/module&gt;
+&lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+        </source>
+        <source>
 @SuppressWarnings({"membername"})
 private int J;
```

Comment:
Please add one more line with class field to show that violation stay if no annotation.
`private int VVV; // violation `
Please update all examples to have code with violation.

Please update existing code without violation to have trailing comment
`// violation suppressed`

Reply:
done